doc/modules/mod_push_service_mongoosepush.md

```diff
@@ -1,39 +1,41 @@
 ### Module Description
 
 This module handles the `push_notification` hook generated by `mod_pubsub` with an active `push` node.
-Each `push_notification` hook is converted as a `REST` API call to the [MongoosePush](https://github.com/esl/MongoosePush) service. 
+Each `push_notification` hook is converted as a `REST` API call to the [MongoosePush](https://github.com/esl/MongoosePush) service.
 The following `publish-options` that are added to the hook are directly passed to `MongoosePush`:
 
  * `mode` - if not supplied, `prod` value will be used
- * `click_action` - optional. 
+ * `click_action` - optional.
  See `click_action` in the FCM documentation or `category` in the APNS documentation.
  * `topic` - _null_ if not supplied
- * `service` - has to be specified and the value must be valid and supported by the MongoosePush push service provider. 
+ * `service` - has to be specified and the value must be valid and supported by the MongoosePush push service provider.
   E.g. `fcm`, `apns`.
  * `device_id` - has to be specified and the value must be valid device token received from push notification service provider specified in `service` option
- 
-In addition to those `publish-options` you may specify also `silent` option, that when set to 
-`true` will result in "silent" notification. 
-Silent notifications send only data payload to push notifications service provider with all 
+
+In addition to those `publish-options` you may specify also `silent` option, that when set to
```

Comment:
In addition to those `publish-options` you may also specify a `silent` option, which, when set to `true`, will result in a "silent" notification.